platform-coop-toolkit/pinecone
# feat(tooling): add clean command, run prior to build

## Description

This PR adds an npm `clean` command to clean up assets, modules, and components prior to a fresh build.

## Steps to test

1. Install updated dependencies: `npm ci`
2. Run build script: `npm run build`
3. Run clean script: `npm run clean`

**Expected behavior:** `/build`, `/dist`, and `/public` are removed.

## Additional information

Not applicable.

## Related issues

Not applicable.
